Added Paperworkbelt to psychologist

SirWarock opened the pull request:
<!-- If you are new to the Delta-V repository, please read the [Contributing Guidelines](https://github.com/DeltaV-Station/Delta-v/blob/master/CONTRIBUTING.md) -->

## About the PR
<!-- What did you change? -->
Added the paperwork belt to the psychologists loadout!

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Weird that a RP role with a lot of paperwork potential doesn't have access to the paperwork belt.

## Technical details
<!-- Summary of code changes for easier review. -->
Added a belt slot for the Psychologist loadout, which fits the paperwork belt.
Added EN translation.

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. -->
![grafik](https://github.com/user-attachments/assets/1be2e0a1-9ad5-48ac-b456-3b394fab6dd4)


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->
None.

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelogs must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- add: Gave the Psychologist a paperwork belt for their loadout!

SirWarock committed:
Added Paperworkbelt to psychologist

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

deltanedas commented on Resources/Locale/en-US/preferences/loadout-groups.ftl:
```diff
@@ -205,6 +205,7 @@ loadout-group-medical-glasses = Medical glasses
 loadout-group-reporter-jumpsuit = Reporter jumpsuit
 
 loadout-group-psychologist-jumpsuit = Psychologist jumpsuit
+loadout-group-psychologist-belt = Psychologist Belt
```
put it in _DV

deltanedas commented on Resources/Prototypes/Loadouts/loadout_groups.yml:
```diff
@@ -1498,6 +1498,12 @@
+  id: PsychologistBelt
+  name: loadout-group-psychologist-belt
+  loadouts:
+  - PaperworkBelt
```
move it to _DV

deltanedas commented on Resources/Prototypes/Loadouts/role_loadouts.yml:
```diff
@@ -553,6 +553,7 @@
   - GroupTankHarness
   - PsychologistJumpsuit
   - MedicalBackpack
+  - PsychologistBelt
```
move it to where the others are below and change the comments to a single begin end pair like in CONTRIBUTING.md

SirWarock committed:
Please the maintainer god

Adding the requested changes and moving some to proper Delta-V namespace, as well as being compliant with CONTRIBUTING.md

SirWarock commented:
HELL YEAH! ORRRRAHH!!
MY FIRST PR MERGED
